node/options_test.go

```diff
@@ -172,18 +172,6 @@ func TestWithBlockChain_ShouldWork(t *testing.T) {
 	assert.Nil(t, err)
 }
 
-func TestWithDataStore_NilStoreShouldErr(t *testing.T) {
```

Comment:
This should not be deleted. You need to fix the assert test and replace node.txSignPrivKey with the store variable

Reply:
done.